src/web/jcc/client/components/RCControllerPanel.tsx

```diff
@@ -147,21 +194,21 @@ export default class RCControllerPanel extends React.Component {
     }
 
     updateRudderDirectionMove(event: IJoystickUpdateEvent) {
-        let bin: { binNumber: number; binValue: number } = { binNumber: 0, binValue: 0 };
+        let rudderBin: Bin = { binNumber: 0, binValue: 0 };
         // The is used to only detect changes if the value is above
         // this percentage (Added when using tablet controller)
         let deadzonePercent = 10;
-        this.calcRudderBinNum(event.x * 100, bin, deadzonePercent);
-        this.props.remoteControlValues.pid_control.rudder = bin.binValue;
+        this.calcRudderBinNum(event.x * 100, rudderBin, deadzonePercent);
+        this.props.remoteControlValues.pid_control.rudder = rudderBin.binValue;
         this.setState({
             rudderDirection: event.direction.toString(),
-            rudderBinNumber: bin.binNumber,
+            rudderBinNumber: rudderBin.binNumber,
         });
     }
 
     calcRudderBinNum(
```

Comment:
We should add a comment for this function since it was changed in this pull request.